public/modules/articles/views/view-article.client.view.html

```diff
@@ -6,7 +6,7 @@ <h1 data-ng-bind="article.title"></h1>
 		<a class="btn btn-primary" href="/#!/articles/{{article._id}}/edit">
 			<i class="glyphicon glyphicon-edit"></i>
 		</a>
-		<a class="btn btn-primary" data-ng-click="remove();">
+		<a class="btn btn-danger" ng-click="return confirm('Are you sure want to remove ?');" data-ng-click="remove();">
```

Comment:
You can't have `ng-click` and `data-ng-click`...one will override the other. Also, stop using confirm(), it is not pretty...Keep all the logic in the controller.